packages/cyberstorm/src/components/Button.tsx

```diff
@@ -19,22 +19,27 @@ export interface ButtonProps {
 /**
  * Cyberstorm Button component
  */
-export const Button: React.FC<ButtonProps> = (props) => {
-  const { label, leftIcon, rightIcon, colorScheme, onClick } = props;
+export const Button: React.FC<ButtonProps> = React.forwardRef(
```

Comment:
IIRC it was discussed that ref forwarding should be reserved for very specific conditions? Now it's hard coded to all Cyberstorm buttons?

Reply:
All components now support ref forwarding. What would be a solution if we want to only use it when needed? 2 different components? :D